downstream/modules/platform/proc-gs-use-base-execution-env.adoc

```diff
@@ -15,19 +32,19 @@ Your subscription with Ansible Automation Platform gives you access to some base
 * `Minimal` - Includes the latest Ansible-core 2.15 release along with Ansible Runner, but does not include collections or other content
 * `EE Supported` - Minimal, plus all Red Hat-supported collections and dependencies
 
-While these environments cover many automation use cases, you can add additional items to customize these containers for your specific needs. For more information about customizing your execution environment, see Customizing an existing automation {ExecEnvShort} image in the Creating and consuming execution environments guide. 
+While these environments cover many automation use cases, you can add additional items to customize these containers for your specific needs. For more information about customizing your execution environment, see link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/{PlatformVers}/html/creating_and_consuming_execution_environments/assembly-publishing-exec-env#proc-customize-ee-image[Customizing an existing automation {ExecEnvShort} image] in the Creating and consuming execution environments guide. 
```

Comment:
Guide name is changed to 'Creating and using execution environments' guide, and use attribute {LinkBuilder} for the guide.

Reply:
updated